packages/commercetools/api-client/src/helpers/commercetoolsLink/linkHandlers.ts

```diff
@@ -20,16 +20,20 @@ export const handleBeforeAuth = async ({ sdkAuth, tokenProvider, apolloReq, curr
   return tokenProvider.getTokenInfo();
 };
 
-export const handleAfterAuth = async ({ sdkAuth, tokenProvider, apolloReq, currentToken }) => {
+export const handleAfterAuth = async ({ sdkAuth, tokenProvider, apolloReq, currentToken, response }) => {
   if (!isUserSession(currentToken) && isUserOperation(apolloReq.operationName)) {
     const { email, password } = apolloReq.variables.draft;
     Logger.debug('Apollo authLinkAfter, customerPasswordFlow', apolloReq.operationName);
 
-    const token = await sdkAuth.customerPasswordFlow({ username: email, password });
-    tokenProvider.setTokenInfo(token);
-    Logger.debug('Apollo authLinkAfter, customerPasswordFlow, generated token: ', getAccessToken(token));
+    if (!response.errors.length) {
```

Comment:
This line causing an error, and because of this go through the auth process is not possible.